### DISCUSSION TITLE
我是不是该关掉windows的音量均衡

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#146] [Q&A 问答]
![无标题](https://user-images.githubusercontent.com/30175425/176670154-41e01dba-4287-4697-a2cb-3d5fc171f612.jpg)

有些视频文件声音很小,音量和正常的视频差很多...
然后我发现开了windows的音量均衡之后,音量都差不多大了.
但我听说这种音量均衡的功能打开之后会破坏音频的动态,把同一个音频里较大的音量压小,较小的音量放大...

**Comment by yeezylife** [comment]
视频文件还算好的(一般同一个音频不会太忽大忽小) 看流媒体尤其直播的时候我都有点离不开这个音量均衡功能了..我尝试打开关闭这个功能用mpv播放了个文件对比了一下,没听出太大区别(当然我不是金耳朵).估计声音大小变化剧烈的文件就会受影响了. 我的想法是如果这个功能不太影响mpv的效果的话...为了便利我开着算了

**Comment by yeezylife** [comment]
这功能还是不能常驻开,我仔细听了听,比如摇滚乐的动态可能比较大. 开了音量均衡之后Ramones的I Don't Wanna Grow Up,开头喊拍子的 "one two three four"音量被明显放大了.和前奏的音量变化明显被压小了...
流媒体和直播声音忽大忽小的问题应该用chrome插件解决...(比如audio chanel,就是每次要自己手动点一下...)

**Comment by hooke007** [comment]
音效方面的问题属于主观题

**Reply by yeezylife** [reply]
我个人看法是不能图简单全局开这个音量均衡功能. 动态压缩太明显.
就是不光是音效变了,连声音的大小关系都剧烈变化了.

**Comment by yeezylife** [comment]
有的视频,人讲话的声音都听不清楚,但是一辆地铁扫过,或者打斗场景,那种环境音又大到刺耳,我觉得这样还是不合理(现实中那声音也没那么大)

我装了个viper音效 我不开任何调音的选项(保持本来的音色), 只开听觉保护,智能音量,和非常轻微/保守的压缩器设置, 在保证平时的人物对话能听清, 声音大小动态关系正常的情况下,不会突然来一声刺耳的巨响...

现在这个状况我比较满意,完全不开(类似功能)的话,有些阴间视频音量动态太炸,然后又不会像windows自带的(音量均衡)那样大幅度的破坏声音的大小关系.
